bitcoin/src/sighash.rs

```diff
@@ -471,6 +472,15 @@ impl fmt::Display for NonStandardSighashType {
     }
 }
 
+impl FromHexStr for NonStandardSighashType {
```

Comment:
Oh, this is an error type so this implementation doesn't make sense.

Reply:
Oh didn't realize that. I'll omit the implementation and make note in the PR